handbook/data-stores/data-stores.md

```diff
@@ -0,0 +1,89 @@
+| | ----------- | ----------- |
+|<img src="../../img/single-box.png" width="1500px"/>| Single node with Defaults     | Here SystemLink and MongoDB are hosted on the same server. Use this configuration when evaluating SystemLink or in small deployments of less than 10 managed nodes without the Asset Manager and Test Insights modules. |
+|<img src="../../img/single-box.png" width="1500px"/>| Single node with increase index cache   | Here SystemLink and MongoDB are hosted on the same server. Use this configuration when evaluating SystemLink or in moderate deployments of less than 25 managed nodes and moderate usage of the Test Insights module producing less than 100 test steps and results each day.     |
+|<img src="../../img/mdb-sa.png" width="1500px" />| Multi node with single standalone MongoDB Instance | Here SystemLink and MongoDB are hosted on two separate servers. Use this configuration when the CPU, memory, and disk consumption is impacting the operation of SystemLink services. Note, while this configuration provides for greater reliability by splitting the servers running SystemLink and MongoDB, it does not provide greater redundancy for data storage.   |
+|<img src="../../img/mdb-rs.png" width="1500px"/>| Multi node with MongoDB replica sets | Here SystemLink is hosted on a dedicated server and MongoDB is hosted on 3 or more servers. **This is the recommended configuration for all production deployments including small and moderately sized deployments.** This configuration enables greater scale (>50 managed nodes, >1000 test steps and results daily). This configuration provides for greater mitigation of data loss with redundant replica sets. Refer to [MongoDB documentation](https://docs.mongodb.com/manual/tutorial/deploy-replica-set/) for setting up and hosting a MongoDB instance with replica sets. |
+|<img src="../../img/mdb-atlas.png" width="1500px"/>| Multi node with MongoDB Atlas | Here SystemLink is hosted on a dedicated server and connects to a MongoDB Atlas cluster. This is recommended for large scale deployments (>200 managed node, 10,000 test steps and results daily). MongoDB Atlas also simplifies database provisioning, operation, backup and restore. These features may be desireable at any scale of deployment. Refer to [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) for details on this service. |
+
+### Changing the index cache size for *single node* deployments
+
+To constrain the amount of memory used by MongoDB, the default installation of SystemLink caps the available memory for the database's index at 1GB. This can lead to high CPU usage if the index grows beyond 1GB and must be paged to disk. If you are using the Test Insights or Asset Management modules, NI recommends increasing the available memory for the index to 8GB. The following steps can be used to increase the available memory for the MongoDB index.
+  
+1. With a user with administrator privileges, log into the desktop of the SystemLink app server.
```

Comment:
Steps look great except they're all step 1 (in this view, at least).

Reply:
That's a common practice in Markdown. They will render as an ordered list in the handbook, but it makes the entire list more maintainable while also obeying our linting rules.